lib/eventbrite_sdk/resource/operations/endpoint.rb

```diff
@@ -55,21 +53,37 @@ def retrieve(params, request = EventbriteSDK)
           # requires those parameters to build the correct resource url path.
           # See the retrieve method (above) for additional details.
           def resource_path(path)
-            @path = path
+            if path.is_a?(Hash)
+              @paths = path
```

Comment:
It appears this will only support create and update. Should we slice out those to be explicit?
`@paths = path.slice(:create, :update)`

Reply:
Sure